x-pack/plugins/ml/common/types/data_frame_analytics.ts

```diff
@@ -59,13 +59,19 @@ export type AnalysisConfig = estypes.MlDataframeAnalysisContainer;
 export interface DataFrameAnalyticsConfig
   extends Omit<estypes.MlDataframeAnalyticsSummary, 'analyzed_fields'> {
   analyzed_fields?: estypes.MlDataframeAnalysisAnalyzedFields;
+  _meta?: Record<string, any>;
 }
 
 export interface UpdateDataFrameAnalyticsConfig {
   allow_lazy_start?: string;
   description?: string;
   model_memory_limit?: string;
   max_num_threads?: number;
+  _meta?: Record<string, any>;
+}
+
+export function isDataFrameAnalyticsConfigs(arg: any): arg is DataFrameAnalyticsConfig {
```

Comment:
we should use `unknown` instead 
```suggestion
export function isDataFrameAnalyticsConfigs(arg: unknown): arg is DataFrameAnalyticsConfig {
```

Reply:
Updated in 642ce6dbe70bb719b6416e5a61184a6b966b71d7